src/python-common/ceph/deployment/service_spec.py

```diff
@@ -389,11 +389,11 @@ def __new__(cls, *args, **kwargs):
         return object.__new__(sub_cls)
 
     def __init__(self,
-                 service_type,     # type: str
-                 service_id=None,  # type: Optional[str]
-                 placement=None,   # type: Optional[PlacementSpec]
-                 count=None,       # type: Optional[int]
-                 unmanaged=False,  # type: bool
+                 service_type: str,
+                 service_id: Optional[str] = None,
+                 placement: Optional[PlacementSpec] = None,
+                 count: Optional[int] = None,
+                 unmanaged: bool = False,
```

Comment:
@jan--f you want to include python-common also in nautilus? Then we have to keep this py2 compatible.